release_config.yml

```diff
@@ -0,0 +1,4 @@
+release-platforms:
+  - GitHub
+  - Maven
```

Comment:
```suggestion
```
I think we don't need a maven central release?

Reply:
Nice finding, yes, not needed to release to maven